src/tests/testString.cc

```diff
@@ -16,7 +16,7 @@ CPPUNIT_TEST_SUITE_REGISTRATION( testString );
 
 /* let this test link sanely */
 void
-eventAdd(const char *name, EVH * func, void *arg, double when, int, bool cbdata)
+eventAdd(const char *, EVH *, void *, double, int, bool)
```

Comment:
should link the test to tests/stub_libevent.cc instead, and remove these lines completely

Reply:
using stub_event.cc instead, and included mem/forward instead of event.h as well